src/contexts/IframeOptionsContext.tsx

```diff
@@ -9,7 +9,7 @@ import { getIsIframe } from '../utils/getIsIframe'
 export const IframeOptionsContext = createContext<{
   isIframe?: boolean
   iframeShareData?: string | null
-  iframeLocalisation?: string | null
+  iframeRegion?: string | null
   iframeOnlySimulation?: boolean
```

Comment:
Quelqu'un m'a dit qu'un booléen commençait toujours par "is", "should" ou "must"

@bjlaa

Reply:
J'ai modifié pour `iframeShareData` et `iframeOnlySimulation` mais `iframeRegion`contient bien un code pays

Reply:
Je parlais de iframeOnlySimulation

Reply:
C'est un oubli de typage yes